hack/test-go.sh

```diff
@@ -8,7 +8,23 @@ set -o pipefail
 
 STARTTIME=$(date +%s)
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
-source "${OS_ROOT}/hack/common.sh"
```

Comment:
It is not removed, it's inside of the if/then case. There are read-only lines in `common.sh` so it cannot be sourced twice.